src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -40,22 +40,21 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+@SuppressWarnings("checkstyle:LineLength")
 /*
  * Suppresses "Line is longer than 120 characters [LineLengthCheck]" warning.  The line length violation was deemed
  * acceptable in this case for the sake of making the table more readable.
- */
-@SuppressWarnings("checkstyle:LineLength")
-/**
- * Maps virtual tables to a set of physical tables hard-coded into the builder by comparing the types of the elements
```

Comment:
Dangling javadoc comment

Reply:
why combine the two comments? changing `/**` to `/*` seems to be what fixed the issue. also, except for that big table, can every line be kept to at most 120 chars? also, can the big table have its axes switched and suddenly fall with the 120-char limit? (not relevant if the presentation isn't the presentation that you feel is necessary.)

Reply:
fixed in 6113e7a9a3f2c33fbef5a66be46dbe1013875fda